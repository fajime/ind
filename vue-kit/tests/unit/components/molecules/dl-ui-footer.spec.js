import { mount } from '@vue/test-utils';

import DlUiFooter from '../../../../src/components/molecules/dl-ui-footer';

describe('dl-ui-footer', () => {

  const wrapper = mount(DlUiFooter);

  it('Component Name', () => expect(DlUiFooter.name).toBe('dl-ui-footer'));

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it('slot content', () => {
    const wrapperTest = mount(DlUiFooter, { slots : { default : '<div class="fake-text"></div>' } });
    const div = wrapperTest.find('div');
    expect(div.findAll('.fake-text').length).toBe(1);
  });

});
