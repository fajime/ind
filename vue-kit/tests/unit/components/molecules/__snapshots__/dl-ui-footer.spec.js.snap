// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dl-ui-footer renders correctly 1`] = `
<footer
  class="dl-ui-footer"
>
  <div
    class="dl-ui-footer__left"
  >
    
    
  </div>
  <div
    class="dl-ui-footer__right"
  >
    <div
      class="dl-ui-footer__img"
    />
  </div>
  <div
    class="dl-ui-footer__right--accent"
  />
</footer>
`;
